tests/LCacheTest.php

```diff
@@ -519,6 +565,7 @@ public function testExistsDatabaseL2()
 
     public function testEmptyCleanUpDatabaseL2()
     {
+        // Why does this test make no assertions?
```

Comment:
I believe it's testing that destructors run properly (without errors/exceptions) with an empty data set. Because all other tests write data, they don't trigger the destuctor in an empty state.